index.d.ts

```diff
@@ -7,6 +7,7 @@ export function client(url: string, options?: Options): Client;
 export function over(socketType: any, options?: Options): Client;
 
 export class Client {
+  connected:boolean;
```

Comment:
While we are exposing `connected`, I suggest we expose also `isBinary`, `subscriptions`, `ws` (because `ws.binaryType` is set in the constructor) and `partialData`.

Also, the spacing should be fixed between the semicolon and the type.